libs/account-lib/src/blockchain/blockchain.service.ts

```diff
@@ -511,4 +510,50 @@ export class BlockchainService implements OnApplicationBootstrap, OnApplicationS
       }
     }
   }
+
+  // eslint-disable-next-line consistent-return
+  public async getRawPayloadForSigning(
+    tx: SubmittableExtrinsic<'promise', ISubmittableResult>,
+    signerAddress: string,
+    // eslint-disable-next-line consistent-return
+  ): Promise<SignerPayloadRaw> {
+    let signRaw;
+    try {
+      await tx.signAsync(signerAddress, {
+        signer: {
+          signRaw: (raw) => {
+            console.log('signRaw called with [raw]:', raw);
+            signRaw = raw;
+            // Interrupt the signing process to get the raw payload, as encoded by polkadot-js
+            throw new Error('Stop here');
+          },
+        },
+      });
+    } catch (_e) {
+      return signRaw;
+    }
+  }
```

Comment:
Thanks a lot, Parity, for making it necessary to do this kind of hokey stuff to get a signing payload 😬 

I think this function can be tweaked as to how the return value is collected so that we don't need to disable lint rules. Also, it can be a static method since it doesn't use any of the class properties (otherwise we could ignore the lint rule for that). Something like:
```suggestion
  public static async getRawPayloadForSigning(
    tx: SubmittableExtrinsic<'promise', ISubmittableResult>,
    signerAddress: string,
  ): Promise<SignerPayloadRaw> {
    const fakeError = '*** Interrupt signing for payload collection ***';
    let signRaw: SignerPayloadRaw;
    try {
      await tx.signAsync(signerAddress, {
        signer: {
          signRaw: (raw) => {
            console.log('signRaw called with [raw]:', raw);
            signRaw = raw;
            // Interrupt the signing process to get the raw payload, as encoded by polkadot-js
            throw new Error(fakeError);
          },
        },
      });
    } catch (e: any) {
      // If we encountered an actual error, re-throw it; otherwise
      // ignore the fake error we threw above
      if (e?.message !== fakeError) {
        throw e;
      }
    }

    return signRaw;
  }
```